pulley/fuzz/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "pulley-interpreter-fuzz"
```

Comment:
This won't get executed on oss-fuzz, how do you feel about that?

If you're ok with it I might recommend a wasm-tools-style approach where the fuzz crate here isn't actually a `cargo fuzz` crate but has a single entrypoint that takes fuzz input and then uses the input to dictate which fuzzer is internally run. Then we'd add a single fuzzer at the crate root for pulley which would internally run all of pulley's fuzzers.

Reply:
Alright, I merged the fuzz targets together into one, like `wasm-tools`.